### ISSUE TITLE
Add the possibility to define font of valueText

### ISSUE THREAD

**Issue by ogarcia** [Mikhus/canvas-gauges#39]
By default valueText uses `/fonts/digital-7-mono.{eot,ttf}`. I think that would be interesting define what font you what to use and were is the fonts. Some like:

```js
valueText: {
    visible: true,
    fontDir: '/fonts'
    font: 'digital-7-mono'
}
```

**Comment by Mikhus** [comment]
Not so bad idea, if you really need that now, feel free to contribute your pull request,or, maybe at a free time me or someone else will implement it...

**Comment by Mikhus** [comment]
Custom fonts are implemented in v2.0.0.

 * [User Guide](https://canvas-gauges.com/documentation/user-guide/)
 * [Tips for Using Custom Fonts](https://canvas-gauges.com/documentation/user-guide/advanced-usage#integration-with-custom-fonts)
